vcd/resource_vcd_rde_type_behavior_acl.go

```diff
@@ -50,6 +50,11 @@ func resourceVcdRdeTypeBehaviorAccessLevelCreate(ctx context.Context, d *schema.
 }
 
 func resourceVcdRdeTypeBehaviorAccessLevelCreateOrUpdate(ctx context.Context, d *schema.ResourceData, meta interface{}, operation string) diag.Diagnostics {
+	// A mutex is required as we use the method SetBehaviorAccessControls which sets and overrides all Access Levels given in the
+	// input. If two or more resources are created/updated at the same time, they would clash with each other.
+	key := "vcd_rde_type_behavior_acl"
```

Comment:
Thinking on a use case where someone may have "a hundred" RDE resources. Do we really need to lock **all** behaviors on ACL changes? No way to narrow down the scope of the lock?

Reply:
You're right, it could be narrowed down to the RDE Type, like
```
vcdMutexKV.kvLock("vcd_rde_type_behavior_acl" + rdeTypeId)
```
I will change it and test it again.